### DISCUSSION TITLE
Sharing GitHub Unfollow Script Automate

### DISCUSSION THREAD

**Discussion by jfullstackdev** [community/community#110053] [Other Feature Feedback, Questions, & Ideas]
Good day, GitHub Community. Just want to share this very simple script to unfollow users you follow but are not following back on GitHub. This removes the need to manually check them. I used this to scan up to 4k of users without hitting the GitHub API rate limiting, check this out :

https://github.com/jdevstatic/github-unfollow-script

**Comment by queenofcorgis** [comment]
@jdevstatic thanks for sharing this and being such a wonderful presence in the community! I've moved this to our Feed category as it the more relevant space for this.
